presto-docs/build

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
should you store it as `presto-docs/bin/build.sh`?

Reply:
I considered that, but it seems overkill for a single build script.